go.mod

```diff
@@ -1,9 +1,6 @@
 module helm.sh/chartmuseum
 
 go 1.23
-
-toolchain go1.23.0
```

Comment:
this is a dependabot bug, see https://github.com/dependabot/dependabot-core/issues/9527 , I will re-add it after merged .